packages/js/src/lib/widget.ts

```diff
@@ -17,7 +17,21 @@ const errorHandler = ErrorHandler.getInstance();
 let surveyRunning = false;
 let setIsError = (_: boolean) => {};
 
+const shouldDisplayBasedOnPercentage = (displayPercentage: number) => {
```

Comment:
I would like to keep having the `widget.ts` for the display logic and rather move this to `triggerSurvey` in actions.ts :-)